src/block/tile/EnderChest.php

```diff
@@ -27,6 +27,19 @@
 
 class EnderChest extends Spawnable{
 
+	protected int $viewerCount = 0;
+
+	public function getViewerCount() : int{
+		return $this->viewerCount;
+	}
+
+	public function setViewerCount(int $viewerCount) : void{
+		if($viewerCount < 0){
+			throw new \InvalidArgumentException($viewerCount . ' is too small of a value');
```

Comment:
You don't need to include the `viewerCount` in the message - it'll be visible in the stack trace.
However, this message should specify that the value can't be negative.